python/ray/air/config.py

```diff
@@ -601,14 +601,21 @@ class CheckpointConfig:
             This attribute is only supported by trainers that don't take in
             custom training loops. Defaults to True for trainers that support it
             and False for generic function trainables.
-
+        checkpoint_keep_all_ranks: If True, will save checkpoints from all ranked
+            training workers. If False, only checkpoint from rank 0 worker is kept.
+        checkpoint_upload_from_workers: If True, distributed workers
+            will upload their checkpoints to cloud directly. This is so large
+            checkpoint files do not need to be transferred to head node for
+            persistence.
     """
 
     num_to_keep: Optional[int] = None
     checkpoint_score_attribute: Optional[str] = None
     checkpoint_score_order: str = MAX
     checkpoint_frequency: int = 0
     checkpoint_at_end: Optional[bool] = None
+    checkpoint_keep_all_ranks: bool = False
+    checkpoint_upload_from_workers: bool = False
```

Comment:
I know you're just being consistent here, but I'm wondering if we can just get rid of (most) of the `checkpoint_` prefixes here. It's the `CheckpointConfig` after all...

Maybe for the new attributes we can get rid of it?

Reply:
I was thinking exactly the same thing man!!
we should do this when we refactor stuff.
let's not update a thousand names with this PR :)